ipad/Constants/AppRemoteAPIConst.swift

```diff
@@ -56,7 +57,7 @@ class RemoteURLManager {
     static var `default` = {
         // Here We Can use Target Flag to customize
         // Switch Env 
-        return RemoteURLManager(.prod)
+        return RemoteURLManager(.local)
```

Comment:
swap this to test for me champ @sam-warren

Reply:
done